vendor/engines/sanger_sequencing/app/models/sanger_sequencing/primer.rb

```diff
@@ -2,7 +2,14 @@
 
 module SangerSequencing
 
-  class Primer
+  class Primer < ApplicationRecord
+    self.table_name = "sanger_sequencing_primers"
+
+    belongs_to :sanger_product
```

Comment:
Why use a `belongs_to` instead of `has_many through`? 

My guess is we'll have the same primers many times in the DB, which won't be a problem because of the volume but we could have unnecessary repetition in our records.

That makes me wonder whether we should use an input like the one you created for the user to select a primer (or enter a new one) that accepts multi selection. That way users don't have to enter the same primer multiple times and only have one record per primer. Does it make sense?

Reply:
yeap makes sense